doc/builders.rst

```diff
@@ -76,6 +76,22 @@ The builder's "name" must be given to the **-b** command-line option of
 
    .. _Qt help: http://qt-project.org/doc/qt-4.8/qthelp-framework.html
 
+.. module:: sphinx.builders.applehelp
+.. class:: AppleHelpBuilder
+
+   This builder produces an Apple Help Book based on the same output as the
+   standalone HTML builder.
+
+   This builder requires Mac OS X 10.6 or higher to function, as it depends on
```

Comment:
The reason I opted to call them directly is that building an Apple Help bundle is mostly just a case of assembling the collection of files in the right places, then running the indexer to add a help index (and optionally, code-signing). This is in contrast to e.g. the situation with Windows HTML Help, where the files need to be assembled as a help project and then processed by the help compiler to make a special archive file. I think it makes more sense for Sphinx to be able to complete all the necessary steps, since there isn't a special tool that does so.

That said, you’re right that it would be good to have some support for either using a different `hiutil` in a different path, or not generating the index at all and leaving it for later.